lib/src/widgets/content_item/content_item.dart

```diff
@@ -27,6 +25,8 @@ import 'package:provider/provider.dart';
 import 'package:interstellar/src/widgets/content_item/content_item_link_panel.dart';
 import 'package:simplytranslate/simplytranslate.dart';
 
+enum PostComponent { title, image, info, body, link }
```

Comment:
Do you think we should allow moving the action buttons around too, or would it be best to leave that out?